apps/src/code-studio/components/ModelCard.jsx

```diff
@@ -40,6 +40,33 @@ const styles = {
   }
 };
 
+class FeatureDetails extends React.Component {
+  static propTypes = {
+    feature: PropTypes.object
+  };
+
+  render() {
+    const {feature} = this.props;
+
+    return (
+      <div>
+        <p style={styles.bold}>{feature.id}</p>
+        <p>{feature.description}</p>
+        <div>
+          Possible Values:{' '}
+          {!feature.values && (
+            <p style={styles.details}>
+              min: {feature.min} max: {feature.max}
```

Comment:
Is this our preferred presentation for ranges across the whole experience?  Or should we try and figure out a unified design?

Reply:
Seems fair.  My only thought is that we could put a comma between them in the third variant...